packages/mask/src/plugins/RealityCards/hooks/useBaseToken.ts

```diff
@@ -0,0 +1,7 @@
+import { FungibleTokenDetailed, useChainId } from '../../../../../web3-shared/evm'
+import { USDC } from '../../Trader/constants'
```

Comment:
Create a USDC constant in this plugin. Importing it from other plugins is not a good idea cause it violates the single responsibility principle.